### DISCUSSION TITLE
Connection reset by peer when pulling image

### DISCUSSION THREAD

**Discussion by piqurs** [podman-desktop/podman-desktop#3247] [Q&A]
Hello, I'm new to Podman and need some assistance. I was trying to pull image from the [Getting Started](https://podman-desktop.io/docs/getting-started/pulling-an-image) page but when I pull, I got this message `https://podman-desktop.io/docs/getting-started/pulling-an-image` 
![image](https://github.com/containers/podman-desktop/assets/42064255/3bfd7454-79e2-4606-bd23-bdec3d031b81)
same result when I pull from the console.
![image](https://github.com/containers/podman-desktop/assets/42064255/a4a88d61-17ea-4826-be3c-7a271c8d8288) I've been searching for the same problem but found no luck. A little help would be appreciate. Thanks in advance.

**Comment by jeffmaury** [comment]
Is it permanent ? Are you behind a proxy

**Reply by piqurs** [reply]
I'm sorry I dont understand what is permanent? Yes I'm behind a proxy. Proxy matters? Can I whitelist it?

**Reply by jeffmaury** [reply]
Did you configure the proxy in Podman Desktop ?

**Reply by piqurs** [reply]
I dont think I have. How do I do that?

**Reply by jeffmaury** [reply]
Go the the Settings page, click Proxy on the left and configure the proxy information

![proxy](https://github.com/containers/podman-desktop/assets/695993/3fbe5d47-062c-407c-a467-6d0e103169b5)

**Reply by piqurs** [reply]
Okay thanks, I will try this. Will come back later after solved. Thanks for the assistance.

**Reply by piqurs** [reply]
I've managed to configure the proxy and when I tried, I got this. `Proxy Authentication Required` Does this mean I need to ask from Security Dept or can just bypass this through podman?
![image](https://github.com/containers/podman-desktop/assets/42064255/84379c50-1327-4d69-b78b-23cf836ee049)

**Reply by piqurs** [reply]
Now it says. Did I miss something? 
![image](https://github.com/containers/podman-desktop/assets/42064255/c9cf5386-2537-40df-9ff3-9e8d2b970703)

**Reply by lstocchi** [reply]
I think you need to start your podman machine

**Reply by piqurs** [reply]
When I checked the podman desktop, it was starting

**Reply by mairin** [reply]
@piqurs any luck?

**Reply by piqurs** [reply]
Nope.
